keras_core/random/seed_generator.py

```diff
@@ -35,8 +35,10 @@ def __init__(self, seed, **kwargs):
         else:
             self.backend = backend
 
+        self.initialized_with_none = seed is None
```

Comment:
Let's use something less verbose like `_initial_seed` (private). Then we can check `is not None`

Reply:
Done.